src/tekton/clustertask.ts

```diff
@@ -16,8 +18,15 @@ export class ClusterTask extends TektonItem {
             if (clustertasks) { ClusterTask.tkn.executeInTerminal(Command.listClusterTasksinTerminal()); }
     }
 
-    static async delete(clustertask: TektonNode): Promise<void> {
-        if (clustertask) { ClusterTask.tkn.executeInTerminal(Command.deleteClusterTask(clustertask.getName())); }
+    static async delete(clustertask: TektonNode): Promise<string> {
```

Comment:
Seems to be good request except some issues with messages:
1. Every message should start with capital letter.
2. Should use appropriate articles a/an/the.
3. Message should finish with './?/!'
4. Custom resource names should be in camel case: ClusterTask, TaskRun, PiplelineResource, PipelineRun, Task, Pipeline

Below is an example:
```
        const value = await window.showWarningMessage(`Do you want to delete the ClusterTask '${clustertask.getName()}\'?`, 'Yes', 'Cancel');
        if (value === 'Yes') {
            return Progress.execFunctionWithProgress(`Deleting the ClusterTask '${clustertask.getName()}'.`, () => 
                ClusterTask.tkn.execute(Command.deleteClusterTask(clustertask.getName())))
                .then(() => `The ClusterTask '${clustertask.getName()}' successfully deleted.`)
                .catch((err) => Promise.reject(`Failure to delete the ClusterTask '${clustertask.getName()}': '${err}'.`));
```